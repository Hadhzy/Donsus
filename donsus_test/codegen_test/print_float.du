my_float: f32 = 12.12;
my_float_2: f64 = 12.12;

printf(my_float, "\n");
printf(my_float_2);