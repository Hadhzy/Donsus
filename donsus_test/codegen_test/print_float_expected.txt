12.120000
12.120000